Topics/Software_Engineering/Scrum Planning Meeting.md

```diff
@@ -0,0 +1,32 @@
+
+The ScrumMaster is responsible for leading the Planning Meeting and the Product Owner is responsible for detailing the Product Backlog items and ensuring alignment with the product’s goals. The planning addresses the value the sprint can bring to the stakeholders and the team collaborates to define a *Sprint Goal* before the end of the Sprint Planning meeting. Following, upon discussion with the Product Owner, the developers select items from the Product Backlog to include in the Sprint Backlog and determine this based on their prior and future Sprint performance. Additionally, for each Sprint Backlog item that is selected, the developers on the team decide how to break down and complete tasks for the sprint based on the team’s definition of “Done”.
+
+
+## Integrating Sprint Planning Meetings in CSC301 
+*Consider an example of how a Scrum Planning Meeting can be integrated into our course’s workflow.*
+
+### Before the Planning Meeting:
+After a Product Manager and Scrum Master are chosen in Deliverable 1, consider each deliverable (deliverable 2 to 5) equivalent to a sprint, with each sprint having one dedicated Sprint Planning Meeting. As a ScrumMaster and Product Manager, organize an hour long meeting within the next week after the Deliverable handout is released. Prior to the Scrum Planning Meeting, you both should also read through the handout and identify features/Product Backlog items for the upcoming sprint and add them to the Product Backlog list on Jira (or the team’s chosen product management tool).
+
+### During the Planning Meeting: 
+Start by reminding the team of the upcoming Deliverable and review the definition of “Done”, which in this case would be meeting both the rubric and your partner's requirements. As a team, take this opportunity to brainstorm more tasks that need to be completed based on what your partner has asked and add them into the Product Backlog list.
+
+Discuss how much time would be needed for each Product Backlog item listed on Jira and based on this estimate, determine items for the Sprint Backlog. Following, select items from the Product Backlog and add them into the Sprint Backlog, allow team members to assign themselves to sprint tasks (including yourself) and discuss any further action items as a team. 
+
+Once the Sprint Backlog items are finalized, as the ScrumMaster, call for a group consensus on the plan and conclude the meeting.
+
+## Key Takeaway
+The Sprint Planning Meeting lays the foundation for the sprint, helps the team understand their tasks and sets the stage for effective Scrum meetings in the weeks ahead. This meeting ensures everyone is aligned and ready for the work to come, making future collaborations smoother and more productive.
+
+## Resources
+- [What is Sprint Planning](https://www.atlassian.com/agile/project-management/project-management-intro)
```

Comment:
This link redirects you to the topic of "Agile vs. waterfall project management".
I think it would be more helpful to provide a link to the direct section on sprint planning:
https://www.atlassian.com/agile/scrum/sprint-planning
This makes it more accessible and convenient for the viewer if they wish to learn more about spring planning

Reply:
Thank you for pointing this out! I will include a link directly to the topic of Sprint Planning from _Scrum.org_.